decision_making.md

```diff
@@ -0,0 +1,45 @@
+
+## Required aspects of decision making
+
+The decision making process is intended to balance broad participation of stakeholders with agility. While the size of decision making bodies may vary (e.g. software projects can range from a handful to dozens of decision makers), all need to find this balance.
+
+All official Jupyter projects, working groups, governing bodies, and official initiatives must have a clearly defined, and documented decision making body. Formally, these decision making bodies make decisions using consensus seeking with an option to call a vote to move the decision forward.
+
+Depending on the governing body, decisions may be proposed by members of the decision making body, or by a broader group of stakeholders/contributors.
+
+**Informal consensus seeking.** Decision making starts with informal consensus seeking. The goal of this discussion phase is to refine the proposal, consider alternatives, weigh trade-offs, and attempt to find informal consensus. The legitimacy of the consensus seeking process is predicated on all stakeholders having their voices heard; teams must be proactive in providing opportunity for all relevant stakeholders to provide input. If the decision making body arrives at informal consensus, they may immediately move to document and enact the decision. This is the consensus seeking phase.
+
+**Calling a vote.** When the discussion matures during the consensus seeking phase, any member of the decision making body can call the matter to a vote. When that member (the sponsor) calls the vote, they shall summarize the proposal in its current form and notify the entire decision making body. After the proposal is seconded by another member of the decision making body, members have 7 days to vote. The governing body may consider longer voting periods as necessary for special circumstances, or _shorter periods only if all voting members are present_. The result will be determined by a simple majority for binary decisions and ranked choice for multiclass decisions (one among many, or several among many). The sponsor may update the proposal at any point during the voting period, in which case, the voting period will be reset.
+
+**Voter participation and quorum.** The various Jupyter decision making bodies often have vast scopes and members are often busy. As a result, there are typically a subset of members who have the time, interest, and expertise to dive into the details of a given decision. We believe that open, transparent, and time-bound voting with no quorum will enable these ad-hoc, informal subgroups to form spontaneously on a per-decision basis, in a manner that supports our goals of responsiveness and engagement.
```

Comment:
```suggestion
**Voter participation and quorum.** The Jupyter decision making bodies often have broad scope. There are typically a subset of members who have the time, interest, and expertise to dive into the details of a limited scope issue, such as subproject specific issues. We believe that open, transparent, and time-bound voting with no quorum will enable these ad-hoc, informal subgroups to form spontaneously for these limited scope issues and work on a per-decision basis, in a manner that supports our goals of responsiveness and engagement.
```

Reply:
I would remove some of the rationale (busy individuals) that make the governance sound weak. Instead, focusing on the specific governance bounds and intent would improve the clarity. Suggested wording above.

Reply:
Thx Carol, merged!